op-deployer/pkg/deployer/opcm/bootstrap_test.go

```diff
@@ -0,0 +1,32 @@
+package opcm_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/foundry"
+	"github.com/ethereum-optimism/optimism/op-chain-ops/script"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/stretchr/testify/require"
+)
+
+// createTestHost is a helper function for testing deploy script wrappers
+func createTestHost(t *testing.T) *script.Host {
+	// Create a logger
+	logger, _ := testlog.CaptureLogger(t, log.LevelInfo)
```

Comment:
Shall we mark this as a [helper function](https://pkg.go.dev/testing#T.Helper)?


```suggestion
func createTestHost(t *testing.T) *script.Host {
    t.Helper()
    
	// Create a logger
	logger, _ := testlog.CaptureLogger(t, log.LevelInfo)
```

Reply:
Oh that's a nice lil helper